acms-react-app/src/Pages/Admin_Pages/CustomerVehicleDetails_Page/CustomerVehicleDetails.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+import Navbar from '../../../Components/Navigation_Bars/Logged_In/NavBar.js';
+import Footer from '../../../Components/Footer/Footer.js';
+import MechanicDisplay from '../../../Components/User_Components/MechanicDisplay.js';
+import Sidebar from '../../../Components/Navigation_Bars/Sidebar/Sidebar.js';
+import './CustomerVehicleDetails.css';
+
+function CustomerVehicleDetails() {
+  const { customerId, vehicleId } = useParams();
+  const [vehicleDetails, setVehicleDetails] = useState({
+    make: '',
+    model: '',
+    year: '',
+    transmission_type: '',
+    mileage: ''
+  });
+
+  function handleInputChange(event) {
+    const { name, value } = event.target;
+    setVehicleDetails(prevState => ({
+      ...prevState,
+      [name]: value
+    }));
+  }
+
+  function updateCustomerVehicle(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const updatedCustomerVehicle = {
+      firstName: formData.get('firstName'),
+      make: formData.get('make'),
+      model: formData.get('model'),
+      year: formData.get('year'),
+      transmissionType: formData.get('transmissionType'),
+      mileage: formData.get('mileage')
+      
```

Comment:
Good work Michael on the create vehicle functionality Michael. Thanks for also fixing my error regarding the transmission setting in the front-end for you. Respect #Rich&Handsome